src/main/webapp/style.css

```diff
@@ -141,3 +141,8 @@
 .trip-button {
   margin-right: 5px;
 }
+
+#map {
+  height: 200px;
```

Comment:
Change the height to `250px`. At `200px`, the "fullscreen" button on the Maps is only partially showing.

`200px` (see the two lines above the little person):
![image](https://user-images.githubusercontent.com/7987279/89065302-4350bb80-d320-11ea-8109-602b172418ab.png)

`250px` (see the fullscreen button above the little person):
![image](https://user-images.githubusercontent.com/7987279/89065368-5fecf380-d320-11ea-87b5-d60412f4233d.png)